examples/with-grafbase/next.config.js

```diff
@@ -4,6 +4,7 @@ const nextConfig = {
   swcMinify: true,
   experimental: {
     appDir: true,
+    runtime: 'experimental-edge',
```

Comment:
Adding this to the root config will currently make all app paths dynamic and revalidate won't be able to be honored currently.

Reply:
Nice 👌

Reply:
Is the mentioned behavior desired? If so it may confuse users as `revalidate` is being configured but not used here.